src/routes/console/project-[project]/store.ts

```diff
@@ -3,7 +3,7 @@ import { page } from '$app/stores';
 import type { Models } from '@appwrite.io/console';
 import type { BarSeriesOption } from 'echarts/charts';
 
-export const project = derived(page, ($page) => $page.data.project as Models.Project);
+export const project = derived(page, ($page) => $page.data.project as Models.Project | any);
```

Comment:
Why does it need the `any`? :thinking: and if it's really necessary would it work with `unknown`?

Reply:
We don't have the new project model which has the disallowPersonalData attribute since we are not using the generated SDK yet. 

@TorstenDittmann asked me to patch it up this way and have all the changes in one single branch ( main ) in this case and he'd generate & release a new console sdk after all changes are in there